website/src/pages/storybook/styles.module.css

```diff
@@ -31,3 +31,8 @@
   width: 100%;
   flex-grow: 1;
 }
+
+.storybookEmptyPage {
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
it doesn't work when using the tailwind class elements like `items-center` and `justify-center` ?

Reply:
Yes